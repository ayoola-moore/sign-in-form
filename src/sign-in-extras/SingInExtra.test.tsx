import React from "react";
import renderer from "react-test-renderer";
import 'jest-styled-components'
import SignInExtra from ".";

it("Sign-in component renders correctly", () => {
  const tree = renderer.create(<SignInExtra />).toJSON();
  expect(tree).toMatchSnapshot();
});
