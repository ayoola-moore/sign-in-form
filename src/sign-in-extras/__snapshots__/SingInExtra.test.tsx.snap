// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sign-in component renders correctly 1`] = `
<div
  className="sc-dlfnbm eZnYTT"
>
  <a
    className="sc-bdfBwQ dZCzPq"
    href="#"
  >
    Forget your password?
  </a>
  <div
    className="sc-hKgILt kwOCTC"
  >
    Don't have an account? 
    <a
      className="sc-bdfBwQ sc-gsTCUz dZCzPq idHLlu"
      href="#"
    >
      Sign up
    </a>
  </div>
  <a
    className="sc-bdfBwQ dZCzPq"
    href="#"
  >
    Resend email confirmation
  </a>
</div>
`;
